internal/servers/controller/handlers/hosts/host_service.go

```diff
@@ -176,7 +176,7 @@ func (s Service) createInRepo(ctx context.Context, scopeId, catalogId string, it
 	}
 	out, err := repo.CreateHost(ctx, scopeId, h)
 	if err != nil {
-		if errors.IsUniqueError(err) || errors.Is(err, errors.ErrNotUnique) {
+		if errors.IsDomainError(err) || errors.IsUniqueError(err) || errors.Is(err, errors.ErrNotUnique) {
```

Comment:
do we still need to check for IsUniqueError() and Is errors.ErrNotUnique?  I think, you can just make sure it's errors.Convert(err) != nil